### PR TITLE
Chapter_11

### PR DESCRIPTION
**Pull request by mirulzaei**

### CODE REVIEW

**Comment by mirulzaei** [comment]
ready

File: app/mailers/application_mailer.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: "from@example.com"
+  default from: "noreply@example.com"
```
Comment (ngocvt-0484) [inline app/mailers/application_mailer.rb:2]: use ENV for "noreply@example.com"

File: app/assets/stylesheets/account_activations.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/account_activations.scss]: delete empty file

File: app/helpers/account_activations_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
+end
```
Comment (ngocvt-0484) [inline app/helpers/account_activations_helper.rb]: delete empty file

**Comment by mirulzaei** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**